### ISSUE TITLE
epoch loss and accuracy

### ISSUE THREAD

**Issue by imranshaikmuma** [udibr/headlines#28]
epoch training accuracy is not going above 60 percentage.
I tried with different optimizers like adam, sgd and learning rates like 0.1, 0.001, 1e-4, 0.5 etc and also with batch size like 16 and 8 and also number of rnn layers 3, 6, 12. 

![image](https://user-images.githubusercontent.com/24683611/29279820-156374bc-80e7-11e7-8394-07c734034b4f.png)

please let me know how to improve accuracy?

**Comment by deepbodra96** [comment]
> 
> 
> epoch training accuracy is not going above 60 percentage.
> I tried with different optimizers like adam, sgd and learning rates like 0.1, 0.001, 1e-4, 0.5 etc and also with batch size like 16 and 8 and also number of rnn layers 3, 6, 12.
> 
> ![image](https://user-images.githubusercontent.com/24683611/29279820-156374bc-80e7-11e7-8394-07c734034b4f.png)
> 
> please let me know how to improve accuracy?

I know it's a very basic question but I am a newbie
Can you tell how do you print accuracy?

**Comment by deepbodra96** [comment]
I know it's a very basic question but I am a newbie
Can you tell how do you print accuracy?
